tracing/src/main/java/com/palantir/tracing/Tracer.java

```diff
@@ -91,8 +91,14 @@ private static boolean shouldObserve(Observability observability) {
         throw new SafeIllegalArgumentException("Unknown observability", SafeArg.of("observability", observability));
     }
 
+    /**
+     * In the unsampled case, the Trace.Unsampled class doesn't actually store a spanId/parentSpanId
+     * stack, so we just make one up (just in time). This matches the behaviour of Tracer#startSpan.
+     * <p>
+     * n.b. this is a bit funky because calling getTraceMetadata multiple times will return different spanIds
+     */
     @Beta
```

Comment:
I think if we're making this public can we drop the @Beta pls?

Reply:
Also will need to make the TraceMetadata interface itself public?

Reply:
fixed it up